spring-pulsar/src/main/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainer.java

```diff
@@ -360,6 +360,11 @@ else if (this.listener != null) {
 									else if (this.containerProperties.getAckMode() == AckMode.BATCH) {
 										this.nackableMessages.add(message.getMessageId());
 									}
+									else {
+										throw new IllegalStateException(
+												"Exception occurred and not negatively acknowledged or handled properly",
```

Comment:
Is there any action we can recommend for the user to take in this case? I guess, what is the outcome in this case (data loss?)